charts/seed-bootstrap/charts/nginx-ingress/templates/controller-service.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: nginx-ingress
+    component: "controller"
+    heritage: Tiller
+    release: addons
+  name: nginx-ingress-controller
+  namespace: garden
+  annotations:
+    dns.gardener.cloud/dnsnames: "{{ .Values.domain }}"
```

Comment:
Remove this and create a `DNSEntry` instead

Reply:
If I create a dns entry myself I need to specify a target ip. This ip is dynamically created with the LB service so I can't know it when the charts are applied. A annotation is the recommended way to create dns records for services according to: https://github.com/gardener/external-dns-management#automatic-creation-of-dns-entries-for-services-and-ingresses

Reply:
In the shoot controller we create a load balancer service, wait for the IP to be populated into the status, and then create a `DNSEntry` with it. It is somewhat confusing if we follow a different approach now in the seed controller. I'm just trying to strive for consistency.

Reply:
Should we change it in the shoot controller then? It seems less work to do it with an annotation.

Reply:
We cannot because we create a `DNSEntry` for the external record whose target is not a load balancer IP of `Service`

Reply:
done in https://github.com/gardener/gardener/pull/3131/commits/62040659fa259ee0aa470b71e897eb4e994768e1